sourcery_analytics/main.py

```diff
@@ -161,8 +163,7 @@ def cli_assess(
         console.print(f"[bold red]Found {count} errors.")
         raise typer.Exit(1)
 
-    console.print("[bold green]Assessment Complete")
-    console.print("[green]No issues found.")
+    console.print("[bold green]Assessment Complete", "[green]No issues found.")
```

Comment:
**suggestion (code_refinement):** Use separate calls for distinct messages in console.print.

Combining two messages into a single `console.print` call without a newline or separator might lead to formatting issues. Consider using two separate `console.print` calls or combining the messages with an appropriate separator for clarity.

```suggestion
    console.print("[bold green]Assessment Complete")
    console.print("[green]No issues found.")
```